modules/helm-installing-helm.adoc

```diff
@@ -8,17 +8,51 @@
 .Prerequisites
 * You have installed Go, version 1.13 or higher.
 
-.Procedure
-. To install Helm, download the Helm binary and install Helm with the `make` command:
+== On Linux
+
+. Download the Helm binary and add it to your path:
 +
 ----
-$ curl https://mirror.openshift.com/pub/openshift-v4/clients/helm/latest
-$ GO111MODULE=on make
+# curl -L https://mirror.openshift.com/pub/openshift-v4/clients/helm/latest/helm-linux-amd64 -o /usr/local/bin/helm
+# chmod +x /usr/local/bin/helm
 ----
 
 . Check the installed version:
 +
 ----
 $ helm version
-version.BuildInfo{Version:"v3.0+unreleased", GitCommit:"a41f66e7be615d8602898e4478dec70e5f60e23c", GitTreeState:"clean", GoVersion:"go1.13.1"}
-----
\ No newline at end of file
+version.BuildInfo{Version:"v3.0", GitCommit:"b31719aab7963acf4887a1c1e6d5e53378e34d93", GitTreeState:"clean", GoVersion:"go1.13.4"}
+----
+
+== On Windows 7/8
+
+. Download the latest link:https://mirror.openshift.com/pub/openshift-v4/clients/helm/latest/helm-windows-amd64.exe[exe file] and put in a directory of your preference.
+. Right click *Start* and click *Control Panel*.
+. Select *System and Security* and then click *System*.
+. From the menu on the left, select the *Advanced systems settings* and click the *Environment Variables* button at the bottom.
```

Comment:
* Not necessary to say "button", so remove "button" and the "the" in front of  `*Environment Variables*`
* Remove "the" in front of `*Advanced systems settings*`

Reply:
Fixed.